src/ProjectFileTools.MSBuild/ProjectFileTools.MSBuild.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
+  <PropertyGroup>
+    <TargetFrameworks>net46</TargetFrameworks>
```

Comment:
Is the intention to make this multi-target? If not, we can change this to `TargetFramework` (rather than `TargetFrameworks`) and drop the condition on whether it's being built for net46 on the ItemGroup below